Makefile

```diff
@@ -124,7 +124,7 @@ $(HADOLINT): | $(TOOLSDIR) ; $(info  install hadolint...)
 
 $(HELM): | $(TOOLSDIR) ; $(info  install helm...)
 	$Q curl -sSfL https://get.helm.sh/helm-$(HELM_VER)-linux-amd64.tar.gz -o - | tar xz -C $(TOOLSDIR) --strip-components=1 linux-amd64/helm 
```

Comment:
why not install helm with script?

$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
$ chmod 700 get_helm.sh
$ ./get_helm.sh

Reply:
also how did the first commit passed CI?

Reply:
The main reason why I decided to use tarball-based installation method is to prevent installing helm globally. My personal preference is to install all supplementary tools for project building/linting to the local path. 
I checked again, and it is possible to use `get_helm.sh` for installation to a local directory, so I updated my PR to use this method.

> also how did the first commit passed CI?
Execution bit is already set on the helm binary after it unpacked from the tarball.  The original version of `make lint-helm` was setting exec bit on the wrong binary (hadolint), but that didn't prevent helm installation from working if that wrong binary exists. Hadolint binary was already in place because `make lint-dockerfile` was called before.